### PR TITLE
Create capability instance

### PR DESCRIPTION
**Pull request by nuxzero**
In this PR created a helper function to create an instance of the Capability.

### CODE REVIEW

File: app/src/main/java/co/omise/android/example/PaymentSettingFragment.kt [hunk 1]
```diff
@@ -39,14 +39,13 @@ class PaymentSettingFragment : PreferenceFragmentCompat() {
                 .findPreference<CheckBoxPreference>(getString(R.string.payment_preference_is_use_specifics_payment_methods_key))?.isChecked
                 ?: false
 
-        PaymentSetting.getPaymentMethodSettingKeysWithSourceTypes(context!!).forEach {
-            preferenceScreen.findPreference<CheckBoxPreference>(it.first)?.apply {
+        PaymentSetting.getPaymentMethodPreferences(context!!).forEach {
```
Comment (Namiii) [inline app/src/main/java/co/omise/android/example/PaymentSettingFragment.kt]: There is no way to get rid of the `!!` over here is there? 🤔

File: sdk/src/main/java/co/omise/android/models/Capability.kt [hunk 2]
```diff
@@ -36,4 +37,24 @@ data class Capability(
...
         override fun type(): Class<Capability> = Capability::class.java
     }
+
+    companion object {
+
+        /**
+         * The helper function for creating an instance of [Capability].
+         * @param allowCreditCard allow to create a [Token] with a credit card or not. Default is true.
+         * @param sourceTypes list of [SourceType] that allow to create a [Source].
+         * @return an instance of [Capability] with specific configuration.
```
Comment (Namiii) [inline sdk/src/main/java/co/omise/android/models/Capability.kt]:
```suggestion
         * The helper function for creating an instance of the [Capability] class.
         * @param allowCreditCard allows users to create a [Token] with or without a credit card. Default is true.
         * @param sourceTypes list of [SourceType] that allow users to create a [Source].
         * @return an instance of [Capability] with a specific configuration.
```

Comment (Namiii) [reply sdk/src/main/java/co/omise/android/models/Capability.kt]: Some small changes that I think would help improve the docs.

Comment (nuxzero) [reply sdk/src/main/java/co/omise/android/models/Capability.kt]: OK, I'll make changes.

Comment (nuxzero) [reply sdk/src/main/java/co/omise/android/models/Capability.kt]: OK, I'll make changes.

File: app/src/main/java/co/omise/android/example/PaymentSettingFragment.kt (hunk 1 above)
Comment (nuxzero) [reply app/src/main/java/co/omise/android/example/PaymentSettingFragment.kt]: Yeah, I think so.

**Comment by nuxzero** [comment]
@Namiii I made changes. Could you have another review please. 🙇

**Pull request merged**